authbridge/demos/aiac-github-issue/keycloak_ops/apply_policy.py

```
@@ -0,0 +1,94 @@
+            client_name = mapping["client"]
+            role_name = mapping["role"]
+
+            if client_name not in client_ids:
+                print(f"  Warning: Client '{client_name}' not found")
+                continue
+
+            client_id = client_ids[client_name]
+            try:
+                add_client_role_to_realm_role_composite(admin, realm, user_role, client_id, role_name)
+                print(f"  ✓ Added client role '{client_name}.{role_name}' to realm role '{user_role}'")
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)